src/Card.php

```diff
@@ -252,7 +252,7 @@ public function addModelActions(Model $model)
      */
     public function addSection(string $title = null, Model $model = null, array $fields = null, bool $useTable = false, bool $useLabel = false)
     {
-        $section = $this->add([$this->cardSection, 'card' => $this], 'Section');
+        $section = CardSection::addTo($this, ['card' => $this], ['Section']);
```

Comment:
```suggestion
        $section = CardSection::addToWithCl($this, [$this->cardSection, 'card' => $this], ['Section']);
```

@ibelar why this change?